### PR TITLE
Enable bridge websockets by default for new installs

### PR DESCRIPTION
**Pull request by damencho**
Depends on https://github.com/jitsi/jitsi-videobridge/pull/1462

### CODE REVIEW

File: debian/jitsi-meet-turnserver.postinst [hunk 1]
```diff
@@ -170,18 +150,14 @@ denied-peer-ip=240.0.0.0-255.255.255.255" >> $TURN_CONFIG
...
-                # If we are not multiplexing we need to change the port in prosody config
-                sed -i 's/"443"/"5349"/g' $PROSODY_HOST_CONFIG
-                invoke-rc.d prosody restart || true
-            fi
+        if [ -f $NGINX_STREAM_CONFIG ] ; then
+            echo "------------------------------------------------"
+            echo ""
+            echo "You have multiplexing enabled, it is recommended to disable it and migrate to using websockets for the bridge channel."
+            echo "The support for sctp data channels is deprecated and will be dropped at some point."
+            echo "How to do it at: https://jitsi.org/multiplexing-to-bridge-ws-howto"
```
Comment (bgrozev) [inline debian/jitsi-meet-turnserver.postinst:158]: This 404s for me

Comment (damencho) [reply debian/jitsi-meet-turnserver.postinst:158]: Cause I need to add it :) It is a work in progress

Comment (damencho) [reply debian/jitsi-meet-turnserver.postinst:158]: And that is a redirect we can edit later if something comes up.

**Comment by bgrozev** [comment]
With this we have TURNS on port 5349? And the docs are going to describe how to move it back to 443 via multiplexing on server name?

**Comment by damencho** [comment]
> With this we have TURNS on port 5349? And the docs are going to describe how to move it back to 443 via multiplexing on server name?

Yes!

File: debian/jitsi-meet-prosody.postrm [hunk 2]
```diff
@@ -45,8 +45,12 @@ case "$1" in
             rm -rf /var/lib/prosody/$JICOFO_AUTH_DOMAIN.*
             rm -rf /var/lib/prosody/$JVB_HOSTNAME.*
 
-            # clean created users
-            rm -rf /var/lib/prosody/`echo $JICOFO_AUTH_DOMAIN | sed -e "s/\./%2e/g"`
+            # clean created users, replace '.' with '%2', replace '-' with '%2d'
```
Comment (JonathanLennox) [inline debian/jitsi-meet-prosody.postrm:48]: Comment is missing the 'e' from the replacement for '.'

File: debian/jitsi-meet-turnserver.postinst (hunk 1 above)
Comment (damencho) [reply debian/jitsi-meet-turnserver.postinst:148]: Tha page is now live.

**Review by bgrozev — APPROVED** [review]

**Pull request merged**